starter-code/ShoppingListVer2/app/src/main/java/ly/generalassemb/drewmahrt/shoppinglistver2/ItemObject.java

```diff
@@ -0,0 +1,48 @@
+package ly.generalassemb.drewmahrt.shoppinglistver2;
+
+/**
+ * Created by Serkan on 25/10/16.
+ */
+
+public class ItemObject {
+    private String mItemName, mDescription, mType, mPrice;
+
+    public ItemObject(String itemName, String description, String price, String type) {
+        mItemName = itemName;
+        mDescription = description;
+        mPrice = price;
+        mType = type;
+    }
+
+    public String getDescription() {
+        return mDescription;
+    }
+
+    public void setDescription(String description) {
+        mDescription = description;
+    }
+
+    public String getItemName() {
+        return mItemName;
+    }
+
+    public void setItemName(String itemName) {
+        mItemName = itemName;
```

Comment:
Totally fine to have setters here, but I just want to make sure you know you don't always need to have setters. Sometimes you don't want an object's values to change after it's created, so you might prefer not to provide any setters.